cfme/containers/image.py

```diff
@@ -39,8 +39,14 @@ class entities(ContainerObjectDetailsEntities):  # noqa
 
 
 @attr.s
-class Image(BaseEntity, Taggable, Labelable, LoadDetailsMixin, PolicyProfileAssignable):
-
+class Image(
+    BaseEntity,
+    CustomButtonEventsMixin,
+    Taggable,
+    Labelable,
+    LoadDetailsMixin,
+    PolicyProfileAssignable,
+):
```

Comment:
**Question** Did black put this on new lines?

Reply:
yes... this is black formatting... more readable.